src/libraries/System.Globalization/tests/DateTimeFormatInfo/DateTimeFormatInfoTests.cs

```diff
@@ -60,6 +60,8 @@ public void DateSeparatorTimeSeparator_Get_ReturnsExpected()
         }
 
         [Theory]
+        // Browser's ICU doesn't support NativeCalendarName
+        [PlatformSpecific(~TestPlatforms.Browser)]
```

Comment:
Instead of ignoring we should assert the actual value

Reply:
the test actually works with current version of ICU.
I am going to strip more from ICU (https://github.com/dotnet/icu/pull/5) but that can be update in a separate PR